src/main/java/mate/controller/drivers/CreateDriverController.java

```diff
@@ -0,0 +1,34 @@
+package mate.controller.drivers;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/drivers/create")
```

Comment:
check `Good URLs naming in your project` in the common mistakes
<img width="519" alt="image" src="https://user-images.githubusercontent.com/61326568/191285314-53ac41e1-7f65-433a-bcf2-04b6e16c0f3f.png">